src/Sylius/Bundle/UiBundle/Resources/translations/messages.en.yml

```diff
@@ -44,10 +44,12 @@ sylius:
         administration: Administration
         administration_dashboard: 'Administration dashboard'
         administration_panel_login: Administration panel login
+        admin_users: Administrators
```

Comment:
Shouldn't you be consistent it naming the keys? Here you have `admin_users` translated to `Administrators` which already smells bad, but moreover [here](http://www.huffingtonpost.com/2014/10/20/spotify-family-plan_n_6014606.html), you actually use administrators in the key, so what's up?